draft-ietf-quic-recovery.md

```diff
@@ -353,14 +353,17 @@ endpoint:
 On the first RTT sample for a network path, the smoothed_rtt is set to the
 latest_rtt.
```

Comment:
Moving this down above "On subsequent RTT samples," and adding that rttvar is also set on the first RTT sample  would help clarify this.